wazo_deployd_client/commands/status.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
<nitpicking>
This client is python3 only. Can you remove the `coding` comment on top.
</nitpicking>

Reply:
you're right, it was a copy/paste from other client